packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/gateway/GatewayRouter.sol

```diff
@@ -43,12 +43,15 @@ abstract contract GatewayRouter is TokenGateway {
     event GatewaySet(address indexed l1Token, address indexed gateway);
     event DefaultGatewayUpdated(address newDefaultGateway);
 
-    function _initialize(address _counterpartGateway, address _defaultGateway)
-        internal
-        virtual
-        override
-    {
-        TokenGateway._initialize(_counterpartGateway, address(0));
+    function _initialize(
+        address _counterpartGateway,
+        address _router,
```

Comment:
Did it so we don't have to override TokenGateway._initialize
It also seemed a bit cleaner to have a different function signature since we expect a different param.